aes-kw/src/kek.rs

```diff
@@ -80,11 +110,20 @@ where
     }
 
     /// AES Key Wrap, as defined in RFC 3394.
-    pub fn wrap(&self, data: &[u8]) -> Result<Vec<u8>, Error> {
+    ///
+    /// The `out` buffer will be overwritten, and must be exactly [`IV_LEN`]
+    /// bytes (i.e. 8 bytes) longer than the length of `data`.
+    pub fn wrap(&self, data: &[u8], out: &mut [u8]) -> Result<()> {
         if data.len() % 8 != 0 {
```

Comment:
include `const SEMIBLOCK_LEN: usize = 8;` and replace 8 with const as required, or 
maybe use something based on `Aes::BlockCipher::BlockSize`?